recipes-tag/mpl-modest-image/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "modestimage" %}
+{% set version = "0.2.0" %}
+{% set file_ext = "tar.gz" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  git_url: https://github.com/NSLS-II-CHX/mpl-modest-image
```

Comment:
why are we looking at their fork rather than upstream?

Reply:
See https://github.com/ChrisBeaumont/mpl-modest-image/issues/10 for the request to tag it upstream. No updates since 2017.